bindings/wasm/src/account/types/signature.rs

```diff
@@ -33,4 +35,16 @@ impl WasmSignature {
   pub fn data(&self) -> Vec<u8> {
     self.0.data().to_vec()
   }
+
+  /// Deserializes a JSON object as `Signature`.
+  #[wasm_bindgen(js_name = fromJSON)]
+  pub fn from_json(json_value: JsValue) -> Result<WasmSignature> {
+    json_value.into_serde().wasm_result()
+  }
```

Comment:
Ideally we should have both complementary `fromJSON` and `toJSON` implementations.

```suggestion
  /// Deserializes a JSON object as `Signature`.
  #[wasm_bindgen(js_name = fromJSON)]
  pub fn from_json(json_value: JsValue) -> Result<WasmSignature> {
    json_value.into_serde().map(Self).wasm_result()
  }
```